src/screens/FeedScreen/Content.js

```diff
@@ -250,35 +258,25 @@ const Content = ({
   return (
     <Pressable
       onLayout={handleHeightContainer}
-      onPress={item?.isBlurredPost ? null : () => onPress()}
-      style={[styles.contentFeed(hasComment), style]}>
+      onPress={isBlurredPost ? null : () => onPress()}
+      style={[styles.contentFeed, style]}>
       <BlurredLayer
         withToast={true}
         onPressContent={handleBlurredContent}
-        isVisible={item?.isBlurredPost}>
+        isVisible={isBlurredPost}
+        containerStyle={{
+          flex: 1
+        }}>
         {message?.length > 0 ? (
           <View>
-            <View
-              style={[
-                styles.containerMainText(handleContainerText().isShort),
-                handleContainerText().container,
-                item?.isBlurredPost
-                  ? {
-                      minHeight: 442
-                    }
-                  : {}
-              ]}>
+            <View style={[styles.containerMainText, handleContainerText().container]}>
               {renderHandleTextContent()}
             </View>
           </View>
         ) : null}
 
         {item && item.post_type === POST_TYPE_POLL ? (
-          <View
-            style={[
-              styles.containerMainText(handleContainerText().isShort),
-              {marginVertical: handleMarginVertical(message)}
-            ]}>
+          <View style={[styles.containerMainText]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Pressable` component now only triggers the `onPress` function if the post is not blurred. The `BlurredLayer` component has been wrapped around the entire content, and its visibility is controlled by the `isBlurredPost` variable. This should help prevent interaction with the content when it's supposed to be blurred. However, make sure that this doesn't interfere with other interactions that should still be possible when the content is blurred.



<!-- This is an auto-generated comment by OSS CodeRabbit -->